src/js/view/controls/components/settings/menu.js

```diff
@@ -98,6 +98,12 @@ export function SettingsMenu(onVisibility, onSubmenuAdded, onMenuEmpty) {
             onVisibility(visible, event);
             settingsMenuElement.setAttribute('aria-expanded', 'true');
             document.addEventListener('click', documentClickHandler);
+
+            if (isDefault && event && event.type === 'enter') {
```

Comment:
@DanFerrer A comment here would be helpful so we know what interactions would result in focusing on the first child (sub-nav item).